src/onnx/parse_topk.cpp

```diff
@@ -0,0 +1,58 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/onnx/checks.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_topk : op_parser<parse_topk>
+{
+    std::vector<op_desc> operators() const { return {{"TopK"}}; }
+
+    std::vector<instruction_ref> parse(const op_desc& /*opd*/,
+                                       const onnx_parser& parser,
+                                       onnx_parser::node_info info,
+                                       std::vector<instruction_ref> args) const
+    {
+        int64_t k = 0;
+        if(args.size() == 2)
+        {
+            auto arg_k = args.at(1)->eval();
+            check_arg_empty(arg_k, "PARSE_TopK: k input must be constant");
```

Comment:
while we are still supporting only constant k values, we should open a separate issue to work on optimizations passes for k. For example, if k=1, we can treat as a max operator, and if k=nelements, then return identity.

Reply:
Those are great ideas. Can you open an issue about these optimizations?

Reply:
Actually I'll need to test the expected behavior since it looks like there are multiple outputs for the op.

Reply:
Yes, it also output the indices of the corresponding elements.